microsoft-edge/web-platform/how-to-detect-win11.md

```diff
@@ -0,0 +1,101 @@
+---
+description: How to differentiate Windows 10 and Windows 11 using User-Agent Client Hints
+title: Detecting Windows 11 using User-Agent Client Hints
+author: MSEdgeTeam
+ms.author: msedgedevrel
+ms.date: 10/15/2021
+ms.topic: article
+ms.prod: microsoft-edge
+keywords: microsoft edge, compatibility, web platform, user-agent string, ua string, user-agent client hints, user agent client hints, ua client hints, ua ch, windows 11, win11, detecting windows 11, windows detection
+---
+# Detecting Windows 11 using User-Agent Client Hints
+
+<!--
+Restrict the lexicon to these forms:
+User-Agent string
+user agent string
+User-Agent Client Hints
+user agent information
+-->
+
+Websites can use user agent information sent from the browser to detect brand, version, device platform, and more. There are two approaches for sites to access user agent information:
+*  User-Agent strings (legacy).
+*  User-Agent Client Hints (recommended).
+
+For details about these two approaches, see [Detecting Microsoft Edge from your website](user-agent-guidance.md).
+
+Starting in Microsoft Edge version 94 and Chrome version 95, sites can differentiate between users on Windows 11 and Windows 10 via User-Agent Client Hints (UA-CH). This information can be found in the following UA-CH request headers:
+
+| Header field | Values that indicate Windows 10 | Values that indicate Windows 11 |
+| --- | --- | --- |
+| `Sec-CH-UA-Platform` | `Windows` | `Windows` |
+| `Sec-CH-UA-Platform-Version` | values between `1.0.0` and `10.0.0` | `13.0.0` and above |
+
+User-Agent strings won't be updated to differentiate between Windows 11 and Windows 10.  We don't recommend using User-Agent strings to retrieve user agent data.  Browsers that do not support User-Agent Client Hints will not be able to differentiate between Windows 11 and Windows 10.
+
+
+<!-- ====================================================================== -->
+## Browsers that support User-Agent Client Hints
+
+The following table shows which browsers support differentiating between Windows 11 and Windows 10.
+
+| Browser | Supports differentiation via User-Agent Client Hints? | Alternative detection methods |
+| --- | --- | --- |
+| Microsoft Edge 94+ | Yes | - |
+| Chrome 95+ | Yes | - |
+| Opera | Yes | - |
+| Firefox | No | None |
+| Internet Explorer 11 | No | None |
+| Microsoft Edge Legacy | No | window.external.getHostEnvironmentValue |
```

Comment:
Let's remove this line entirely. Support for Edge Legacy ended on March 9, 2021. We should not document its support/non-support for things at this point. I briefly thought there might be some applicability for WWA apps using the Edge Legacy platform, but the getHostEnvironmentValue API was never available to those.